<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCCITT4Client</class>
 <widget class="QMainWindow" name="CCCITT4Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CCCITT4 Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/CCITT4Client.qrc">
    <normaloff>:/new/prefix1/logo_tiff_top_largest.png</normaloff>:/new/prefix1/logo_tiff_top_largest.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>6</number>
         </property>
         <widget class="QWidget" name="widgetViewer" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitterWidgetViewer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="handleWidth">
              <number>6</number>
             </property>
             <widget class="QWidget" name="widgetImageViewer" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="CMyGraphicsView" name="graphicsView"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widgetRxLog" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextEdit" name="textEditRxLog">
                 <property name="font">
                  <font>
                   <family>TlwgMono</family>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="tabStopWidth">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widgetDataList" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="widgetFileBrowser" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widgetLogo" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image: url(:/new/prefix1/logo_tiff_top_largest.png);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidgetFiles"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetFilePath" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Storage folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtPath">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>48</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="BtConnect">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Connect</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/CCITT4Client.qrc">
           <normaloff>:/new/prefix1/Disconnect.png</normaloff>
           <normalon>:/new/prefix1/Connect2.png</normalon>:/new/prefix1/Disconnect.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtSingleShot">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/CCITT4Client.qrc">
           <normaloff>:/new/prefix1/Basic 2/PlayPause.png</normaloff>:/new/prefix1/Basic 2/PlayPause.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtRepeat">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/CCITT4Client.qrc">
           <normaloff>:/new/prefix1/Basic 2/Play.png</normaloff>
           <normalon>:/new/prefix1/Basic 2/Stop.png</normalon>:/new/prefix1/Basic 2/Play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">CMyGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/CCITT4Client.qrc"/>
 </resources>
 <connections/>
</ui>
